resources/views/backend/dashboard.blade.php

```diff
@@ -5,133 +5,117 @@
         <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:400,700|Material+Icons" />
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/latest/css/font-awesome.min.css">
         <link rel="stylesheet" href="https://demos.creative-tim.com/material-dashboard/assets/css/material-dashboard.min.css?v=2.1.2">
-  
+        <link rel="stylesheet" href="/css/dash.css" />
```

Comment:
Please use `asset()` so the link doesn't break when on the server

Reply:
Done